app/assets/javascripts/scp-react-rails.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
I would call this file 'bulkDownloadClipboard.js' and add a line at the beginning of the file like 
`window.SCP = window.SCP || {}`
that way we don't have to declare all our SCP globals in the same file, but also don't have to worry about loading order

Reply:
Per our chat abouth this, in 619e805 I did a `git mv` of this file from `scp-react-rails.js` to `react_rails/bulkDownloadClipboard.js` and updated to use that technique.